src/root/body/index.tsx

```diff
@@ -16,20 +15,32 @@ import SignUp from "../../components/body/authentication/signup";
 import CreativeCommonsPopup from "../../components/body/creative_commons_popup";
 import JSZip from "jszip";
 import axios from "axios";
+import UTIF from "utif";
 
 interface ImageCache {
   index: number;
   src: string;
   scores: number[];
   classifications: string[];
-  boxes: any[];
+  boxes: Array<{
```

Comment:
I'm not quite sure that boxes will have that look as it could also contains the seed scores and labels.

[nachet-model-documentation](https://github.com/ai-cfia/nachet-backend/blob/51-implementing-2-models/docs/nachet-model-documentation.md#return-value-of-models)

Reply:
It'll probably need to be verified since this method does not follow that pattern 
```JavaScript
 if (switchTable) {
                      ctx.fillText(
                        `[${labelIndex + 1}] - ${scorePercentage}%`,
                        ((storedImage.boxes[index].bottomX as number) -
                          (storedImage.boxes[index].topX as number)) /
                          2 +
                          (storedImage.boxes[index].topX as number),
                        (storedImage.boxes[index].bottomY as number) + 23,
                      );
```

Reply:
I was able to match model documentation by defining an interface for API call. The "boxes" in ImageCache is a different entity entirely which is why it doesn't match the backend model.

Reply:
Great :)